<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogMain</class>
 <widget class="QDialog" name="DialogMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QT Image Printer</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>File Extension</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="comboBoxFileTypes">
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="currentText">
      <string>*.*</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTreeView" name="treeViewDirectories"/>
     <widget class="QTreeView" name="treeViewSourceFiles">
      <property name="selectionMode">
       <enum>QAbstractItemView::MultiSelection</enum>
      </property>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="pushButtonPrint">
     <property name="text">
      <string>Print</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="comboBoxFilePrinters"/>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="pushButtonAddFiles">
     <property name="text">
      <string>Add Files...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="pushButtonRemove">
     <property name="text">
      <string>Remove</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="lineEditFileExtensions">
     <property name="text">
      <string>*.*</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
